helpers/addresses.py

```diff
@@ -44,6 +44,7 @@
         "ops_deployer4": "0xef42D748e09A2d9eF89238c053CE0B6f00236210",
         "ops_deployer5": "0xC6a902de22b10cb176460777ce6e7A12A6b6AE5a",
         "ops_deployer6": "0x7c1D678685B9d2F65F1909b9f2E544786807d46C",
+        "ops_deployer7": "0x7140B5254d80154f9Fc5F86054efB210f3a1e2c6",
```

Comment:
- funded twice by an owner of techops (`0xC69Fb085481bC8C4bfF99B924076656305D9a25D`)
- deployed digg voting shares contract
- confirmed over discord